Build/BuildConfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "BuildVersion": "21.6",
```

Comment:
Is there a case where this is different from `repoVersion` from the AL-Go settings?

Reply:
I removed this from the buildconfig, so we read it from the AL-GO settings instead